pkg/runner/local_docker.go

```diff
@@ -209,17 +209,17 @@ func (*LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wr
 				"testground.runid":    input.RunID,
 			},
 		}
-		artifactDir := filepath.Join(runDir, name)
-		err = os.Mkdir(artifactDir, 0777)
+		assetsDir := filepath.Join(runDir, name)
```

Comment:
nitpick: Maybe rename to `hostAssetsDir`, so that it is clear that this is the directory on the host, and not inside the container.